normalize_repos/skills.json

```diff
@@ -0,0 +1,17 @@
+{
+  "ccos-scripts": [
+    "Python"
+  ],
+  "vocabulary": [
+    "Sass",
+    "JavaScript/Storybook"
+  ],
+  "vue-vocabulary": [
+    "JavaScript/Vue",
+    "JavaScript/Storybook"
+  ],
```

Comment:
RIP `vue-vocabulary`. This does make me wonder—how does this script react to a missing or archived repository? As long as it doesn't block or end the script prematurely I don't think it is a big issue.

Reply:
Nevertheless, fixed in 2331482..